### PR TITLE
Fix wrong button order (Apply and Cancel) in ManageProtectedTermsDialog.

### PR DESCRIPTION
**Pull request by dextep**
Hi, I noticed that 'manage protected terms files' dialog button order was wrong - 'Apply' was on right side.

image

- [x] Change in CHANGELOG.md described (if applicable)
- [ ] Tests created for changes (if applicable)
- [x] Manually tested changed features in running JabRef (always required)
- [x] Screenshots added in PR description (for UI changes)
- [ ] [Checked documentation](https://docs.jabref.org/): Is the information available and up to date? If not created an issue at <https://github.com/JabRef/user-documentation/issues> or, even better, submitted a pull request to the documentation repository.

### CODE REVIEW

File: src/main/java/org/jabref/gui/protectedterms/ManageProtectedTermsDialog.java [hunk 1]
```diff
@@ -3,12 +3,9 @@
 import javax.inject.Inject;
 
 import javafx.fxml.FXML;
-import javafx.scene.control.ButtonType;
```
Comment (koppor) [inline src/main/java/org/jabref/gui/protectedterms/ManageProtectedTermsDialog.java:6]: Please check your IDE setup. It seems that you did not import the JabRef code style. --> https://devdocs.jabref.org/getting-into-the-code/guidelines-for-setting-up-a-local-workspace

File: src/main/java/org/jabref/gui/protectedterms/ManageProtectedTermsDialog.fxml [hunk 2]
```diff
@@ -31,6 +31,6 @@
             </HBox>
         </VBox>
     </content>
+    <ButtonType fx:id="applyButton" buttonData="OK_DONE" text="%Apply" />
```
Comment (koppor) [inline src/main/java/org/jabref/gui/protectedterms/ManageProtectedTermsDialog.fxml:34]: The order of buttons is OS depdenend. @tobiasdiez Does one have apply a special CSS class here? It does not seem to be applied correclty here (refs https://docs.oracle.com/javase/8/javafx/api/javafx/scene/control/ButtonType.html#APPLY)

Comment (dextep) [reply src/main/java/org/jabref/gui/protectedterms/ManageProtectedTermsDialog.fxml:34]: I have checked another dialog class `Customize EntryTypeDialog` and i wanted to keep the pattern, but right now i checked it again and seems to be unused.

Comment (tobiasdiez) [reply src/main/java/org/jabref/gui/protectedterms/ManageProtectedTermsDialog.fxml:34]: The order is displayed here: https://docs.oracle.com/javase/8/javafx/api/javafx/scene/control/ButtonBar.html

ButtonType = APPLY is indeed wrong, as this is for a button that saves the current state but leaves the dialog open. OK_DONE is correct. I would maybe change the display text to `%Save` to make it clearer and prevent confusion.

Comment (tobiasdiez) [reply src/main/java/org/jabref/gui/protectedterms/ManageProtectedTermsDialog.fxml:34]: Yes the id can be safely deleted. It's only needed if the code uses `lookupButton`

File: src/main/java/org/jabref/gui/protectedterms/ManageProtectedTermsDialog.java (hunk 1 above)
Comment (dextep) [reply src/main/java/org/jabref/gui/protectedterms/ManageProtectedTermsDialog.java:6]: That's true. Thanks @koppor!

**Comment by Siedlerchr** [comment]
Thank you very much for your contribution!

**Pull request merged**